nilearn/input_data/nifti_masker.py

```diff
@@ -72,6 +72,14 @@ class NiftiMasker(BaseMasker, CacheMixin):
         to fine-tune mask computation. Please see the related documentation
         for details.
 
+    mask_time : Any type compatible with numpy array indexing
+        Mask for the niimgs along time/4D dimension. Thus complements
+        3D masking by mask_img argument. Data preprocessing will ignore
+        the time mask, whereas NiftiMasker.transform will only consider
+        time-masked images. Use case: Avoid creation of a temporary 4D data
+        nifti, potentially expensive in memory, before application of
+        NiftiMasker.
```

Comment:
> Data preprocessing will ignore the time mask whereas NiftiMasker.transform will only consider time-masked images.

I thought this wasn't true anymore, i.e. the indexing was done before the other treatments like filtering.

> Use case: Avoid creation of a temporary 4D data nifti, potentially expensive in memory, before application of NiftiMasker.

Not sure what you mean by that, it seems to be that indexing does create a temporary 4D image but maybe I am missing something.